shared/message/motion/ServoCommand.proto

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2022 NUbots <nubots@nubots.net>
+ */
+
+syntax = "proto3";
+
+package message.motion;
+
+import "google/protobuf/timestamp.proto";
+
+message ServoState {
+    /// Proportional gain. In proportion to the servo's position error. Gain
+    /// values are in range 0-100.
+    float gain = 1;
+    /// Used to set the servo on or off. Typically either 0 (off) or 100 (on)
+    float torque = 2;
+}
+
+message ServoCommand {
```

Comment:
Why are we adding a second message called `ServoCommand` should this be called something else?

Reply:
One is for interacting with the Director and one is for interacting with Controller.
Eventually the controller one should be deleted and only this remain.

Reply:
Comment this please.

Reply:
Should the comment go here or in the controller one. Because eventually the comment will make no sense once the other message is removed

Reply:
It should go on both, then the one you remove will tell you to remove the comment on the other.